OSDOCS#17254: CQA updates for the CLI tutorial (#103706)
openshift/openshift-docs: modules/getting-started-cli-load-data-output.adoc

[id="getting-started-cli-load-data-output_{context}"]
= Loading data into the database

[role="_abstract"]

> After you have deployed the `mongodb-nationalparks` database, you can load the national park location information into the database.

Reviewer: ```suggestion
After you have deployed an application database, you can load the information that your application needs into the database.
```
